package.json

```diff
@@ -62,7 +62,7 @@
     "@babel/runtime": "^7.23.6",
     "@braintree/sanitize-url": "^6.0.4",
     "@popperjs/core": "^2.11.5",
-    "@react-aria/focus": "^3.16.1",
+    "@react-aria/focus": "^3.12.1",
```

Comment:
We can do just `^3` if you want.

Reply:
Seems a bit too broad but if the `FocusScope` thing works exactly the same then I'm all for it.